crates/uv-fs/src/lib.rs

```diff
@@ -374,6 +384,8 @@ impl Drop for LockedFile {
                 self.0.path().display(),
                 err
             );
+        } else {
+            debug!("Released lock at `{}`", self.0.path().display());
```

Comment:
Should this be `trace!` like the above?

Reply:
Maybe since it comes after all the other output.